Source/WebKit/UIProcess/ios/WKKeyboardScrollingAnimator.mm

```diff
@@ -428,12 +429,29 @@ - (void)willStartInteractiveScroll
     [self stopDisplayLink];
 }
 
+#if HAVE(CORE_ANIMATION_FRAME_RATE_RANGE)
+static CAFrameRateRange highFrameRateRange()
+{
+    static CAFrameRateRange highFrameRateRange = CAFrameRateRangeMake(80, 120, 120);
+    return highFrameRateRange;
+}
+
+static CAHighFrameRateReason highFrameRateReason = CAHighFrameRateReasonMake(44, 2);
```

Comment:
Can we put these magic numbers in a header as named constants?

Reply:
Wrapped all of this in `WebKit/Platform/cocoa/CAFrameRateRangeUtilities.h`.